lib/web/resources_test.go

```diff
@@ -429,6 +435,311 @@ func TestListResources(t *testing.T) {
 	}
 }
 
+func TestListResourcesPing(t *testing.T) {
```

Comment:
Maybe a brief godoc with some background for this test? It looks weird by itself.